refresh images

szumaster1 opened the pull request.

Commit:
Merge pull request #1 from szumaster/szumaster-img

update img of control section in game guides

Commit:
Merge branch '2009scape:master' into master

Commit:
Add files via upload

Commit:
refresh images

+ delete outdated info about spirit tree

ryannathans commented on site/game_guide/how_do_i_get_started0.html:
```diff
@@ -314,22 +314,18 @@ <h1 class="plaque">
-implemented yet! </p>
-<img alt="" height="150" src="../img/main/kbase/getting_started/treegnome.PNG" width="350"/>
-<p><b> Note: This will be removed once the quest is implemented </b></p>
+<p> Just like the Gnome Gliders, you may travel from any spirit tree directly to any other Spirit Tree in the game. </p>
```
might want to mention the required tree gnome village quest

ryannathans commented on site/game_guide/how_do_i_get_started0.html:
```diff
@@ -314,22 +314,18 @@ <h1 class="plaque">
 <p><b> Spirit Trees </b></p>
-<p> Just like the Gnome Gliders, the Spirit Trees are unlocked as of right now because Tree gnome village quest is not
-implemented yet! </p>
-<img alt="" height="150" src="../img/main/kbase/getting_started/treegnome.PNG" width="350"/>
```
delete the old image too instead of just removing the reference to it

Commit:
requested changes

ryannathans commented on site/game_guide/how_do_i_get_started0.html:
```diff
@@ -318,7 +318,7 @@ <h1 class="plaque">
 <br/>
 <p><b> Spirit Trees </b></p>
-<p> Just like the Gnome Gliders, you may travel from any spirit tree directly to any other Spirit Tree in the game. </p>
+<p> Just like the Gnome Gliders, you may travel from any spirit tree directly to any other Spirit Tree in the game, but in this case, it is required to complete the Tree Gnome Village quest!</p>
```
, but first, you must complete the Tree Gnome Village Quest to gain access.

Commit:
correct spelling

https://github.com/2009scape/2009scape.github.io/pull/179#discussion_r1004523589
